testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeTablePropertiesCompatibility.java

```diff
@@ -0,0 +1,82 @@
+import static io.trino.tests.product.TestGroups.PROFILE_SPECIFIC_TESTS;
+import static io.trino.tests.product.deltalake.util.DeltaLakeTestUtils.DATABRICKS_COMMUNICATION_FAILURE_ISSUE;
+import static io.trino.tests.product.deltalake.util.DeltaLakeTestUtils.DATABRICKS_COMMUNICATION_FAILURE_MATCH;
+import static io.trino.tests.product.deltalake.util.DeltaLakeTestUtils.dropDeltaTableWithRetry;
+import static io.trino.tests.product.deltalake.util.DeltaLakeTestUtils.getTablePropertyOnDelta;
+import static io.trino.tests.product.deltalake.util.DeltaLakeTestUtils.getTablePropertyOnTrino;
+import static io.trino.tests.product.utils.QueryExecutors.onDelta;
+import static java.lang.String.format;
+import static org.testng.Assert.assertEquals;
+
+public class TestDeltaLakeTablePropertiesCompatibility
+        extends BaseTestDeltaLakeS3Storage
+{
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS})
+    @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
+    public void testCreateTableWithTablePropertiesOnDelta()
+    {
+        String tableName = "test_dl_create_table_properties_" + randomNameSuffix();
+        String tableDirectory = "databricks-compatibility-test-" + tableName;
+
+        onDelta().executeQuery(format("CREATE TABLE default.%s (col INT) USING DELTA LOCATION 's3://%s/%s' TBLPROPERTIES ('foo'='bar', 'Foo'='Baz')",
+                tableName,
+                bucketName,
+                tableDirectory));
+
+        try {
+            assertEquals("bar", getTablePropertyOnTrino("default", tableName, "foo"));
+            assertEquals("Baz", getTablePropertyOnDelta("default", tableName, "Foo"));
+            assertEquals("bar", getTablePropertyOnDelta("default", tableName, "foo"));
+            assertEquals("Baz", getTablePropertyOnTrino("default", tableName, "Foo"));
+            assertEquals(getTablePropertyOnDelta("default", tableName, "delta.minReaderVersion"), getTablePropertyOnTrino("default", tableName, "delta.minReaderVersion"));
+            assertEquals(getTablePropertyOnDelta("default", tableName, "delta.minWriterVersion"), getTablePropertyOnTrino("default", tableName, "delta.minWriterVersion"));
+        }
+        finally {
+            dropDeltaTableWithRetry("default." + tableName);
+        }
+    }
+
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS})
+    @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
+    public void testCreateTableWithTableFeaturesOnDelta()
+    {
+        String tableName = "test_dl_create_table_features_" + randomNameSuffix();
+        String tableDirectory = "databricks-compatibility-test-" + tableName;
+
+        onDelta().executeQuery(format("CREATE TABLE default.%s (col INT) USING DELTA LOCATION 's3://%s/%s' TBLPROPERTIES ('delta.minReaderVersion'='3', 'delta.minWriterVersion'='7', 'delta.columnMapping.mode'='id')",
```

Comment:
This line is too long. I would wrap before `TBLPROPERTIES`.

Reply:
> `@Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS})`

7 writer version is supported in old Databricks versions if I remember correctly.

Reply:
Reminder.